### ISSUE TITLE
When re-relaunch?

### ISSUE THREAD

**Issue by Nitrooo** [Spare-Network/spare-blockchain#24]
The relaunch has clearly been a complete failure. The blockchain is corrupted and nobody is able to sync and farm. So, when re-relaunch?

**Comment by ghost** [comment]
we are syncing check your settings... bra

**Comment by kc-sudo** [comment]
Seems the code was updated recently. folder naming changed from .sparecoin to spare-coin. Nothing works now

**Comment by katzenhai2** [comment]
> 
> 
> we are syncing check your settings... bra

There are two blockchains active: The old one before the relaunch and the new one. The correct height now is ~1,066 and the old one was ~27,xxx. I get proofs but no rewards in my wallet. Seems no one else get rewards. Also dev seems not to have a look into the issues here. Otherwise he would have noticed it long before and also the solution for this (its written in there).

**Comment by katolla** [comment]
relaunch already happened, but everyone who uses latest github version is stuck at 662, older versions switch between old and new chain, relaunch failed!

**Comment by katolla** [comment]
> Seems the code was updated recently. folder naming changed from .sparecoin to spare-coin. Nothing works now

yes that update has broken everything

**Comment by ghost** [comment]
no it did not we are using new version and are synced at 1002 +

**Comment by the-vibe-dev** [comment]
I am using new version and not syncing, something is clearly broken.

**Comment by walruzperil** [comment]
Still no sync... So, when relunch?))

**Comment by djjaeger82** [comment]
> no it did not we are using new version and are synced at 1002 +

which new version, there have been 2 code updates after the 3pm utc relaunch.... those of us using the latest (released on github 1hr ago for linux only so far) are all stuck at 662

**Comment by ghost** [comment]
go here https://github.com/Spare-Network/spare-blockchain/issues/14

get some of those ips in.

**Comment by joe-essenfeld** [comment]
Things should stabilize once the block height of the relaunch exceeds the
block height of the original launch.

**Comment by djjaeger82** [comment]
> go here #14
> 
> get some of those ips in.

an IP list from 2 hrs ago, before the post relaunch code updates?  won't help.... you're out of date.  If you think you're synced at 1xxx you're still on an old code base where 2 chains are running in parallel screwing everything up

**Comment by the-vibe-dev** [comment]
Yes the new chain with the updated certs is not syncing.

**Comment by Nitrooo** [comment]
> no it did not we are using new version and are synced at 1002 +

You can keep saying "this is fine, this is fine, everything is fine", but reality is that this is a broken hiddenly pre-farmed copycat shitcoin ran by some random guy who has never seen a line of code in his entire life.

So, when re-relaunch?

**Comment by kc-sudo** [comment]
> > go here #14
> > get some of those ips in.
> 
> an IP list from 2 hrs ago, before the post relaunch code updates? won't help.... you're out of date. If you think you're synced at 1xxx you're still on an old code base where 2 chains are running in parallel screwing everything up

You are right. Anything older than an hour is messed up. Those old IPs don't work. New update chain still stuck at 662.

**Comment by djjaeger82** [comment]
Anyone else on latest github (5e7841b from ~2hr ago) suddenly in sync?  Previously was stuck at 662 but seems to be iterating.  Looks like all of us are now at 1078...

**Comment by the-vibe-dev** [comment]
I am on the latest github but still cannot sync, can you post your node list?

**Comment by djjaeger82** [comment]
Well it only stayed synced for about 3min.  Ran from 662 up to 1078, now we're all stuck on 1078 with latest github it looks like :(

**Comment by the-vibe-dev** [comment]
I have 0 nodes connected with the newest update....

**Comment by Namsoyo** [comment]
> I have 0 nodes connected with the newest update....

same, is there a new port?

**Comment by root-spare** [comment]
Please get the latest code , most of issues are resolved now

**Comment by kc-sudo** [comment]
Is this a joke? Now folder namings have changed from .spare-coin to .spare-blockchain. this is the third change in two hrs. Seems this is becoming a joke

**Comment by Puritosfaria** [comment]
> Please get the latest code , most of issues are resolved now

I did that and i cant even sync now... xDDDD Will wait, ty anyway.
